.circleci/config.yml

```diff
@@ -1,8 +1,434 @@
+      study_guid:
+        type: string
+    steps:
+      - checkout:
+          path: ~/repo
+      - setup-shared-env:
+          study_key: << parameters.study_key >>
+      - set-deployment-environment
+      - run:
+          name: Retrieve and expand build archive for << parameters.study_key >> with git SHA << pipeline.git.revision >>
+          command: |
+            set -u
+            # we are getting our builds from the one bucket
+            readvault.sh secret/pepper/prod/v1/conf .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
+            TAR_FILE_URL_PATTERN="gs://${BUILDS_BUCKET}/${ANGULAR_PROJECT_NAME}/${ANGULAR_PROJECT_NAME}_*_${SHORT_GIT_SHA}.tar.gz"
+            TAR_FILE_URL=`gsutil ls $TAR_FILE_URL_PATTERN  | sort -r | head -1`
+
+            if [ -z $TAR_FILE_URL ]
+            then
+              echo "Could not find archive using pattern ${TAR_FILE_URL_PATTERN}"
+              exit 1
+            fi
+            TAR_FILE_PATH="/tmp/$(basename -- $TAR_FILE_URL)"
+            gsutil cp  $TAR_FILE_URL $TAR_FILE_PATH
+            cd ~/repo/ddp-workspace
+            mkdir -p $OUTPUT_DIR && tar -xvf $TAR_FILE_PATH -C $OUTPUT_DIR
+            echo "Contents of extracted archive at $OUTPUT_DIR"
+            ls -l $OUTPUT_DIR
+      - deploy-exploded-build:
+          study_key: << parameters.study_key >>
+          study_guid: << parameters.study_guid >>
+
+  deploy-exploded-build:
+    description: Deploy files to GAE
+    parameters:
+      study_key:
+        type: string
+      study_guid:
+        type: string
+    steps:
+      - set-deployment-environment
+      - run:
+          name: <<parameters.study_key>> generate config files and copy pepperConfig.js
+          command: |
+            set -u
+            ./build-study.sh v1 $ENVIRONMENT . <<parameters.study_key>> <<parameters.study_guid>> --config
+            cd ddp-workspace
+            CONFIG_DIR="${OUTPUT_DIR}/assets/config"
+            mkdir -p $CONFIG_DIR
+            CONFIG_FILE_SRC_PATH="${ANGULAR_PROJECT_DIR_PATH}/output-config/pepperConfig.js"
+            cp  $CONFIG_FILE_SRC_PATH $CONFIG_DIR
+            echo "$CONFIG_FILE_SRC_PATH has been copied to $CONFIG_DIR"
+          environment: #Setting this ENV variable prevents launching a new Docker within build-study-sh script
+            USE_DOCKER: false
+      - run:
+          name: Setup gcloud context
+          command: |
+            set -u
+            readvault.sh secret/pepper/${ENVIRONMENT}/v1/conf .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
+            gcloud --quiet config set compute/zone ${GOOGLE_COMPUTE_ZONE}
+          working_directory: ~/repo
+      - run:
+          name: Update GAE dispatch file
+          command: |
+            set -u
+            gcloud app deploy --project "broad-ddp-${ENVIRONMENT}" "${ANGULAR_PROJECT_DIR_PATH}/output-config/dispatch.yaml"
+          working_directory: ~/repo/ddp-workspace
```

Comment:
This chunk was picked up from previous hotfix @zyme can you confirm? And do we need to bring this to `develop`?

Reply:
Yes, we need to bring this over as well.